python/tests/scripts/e2_states.metta

```diff
@@ -1,25 +1,35 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+; State atoms
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+; `new-state` creates a new state atom wrapping the initial value (A B).
+; Then it binds this state atom to the variable `&state-token`.
```

Comment:
It is not a variable. It is actually a token, which is replaced by the corresponding atom at parse-time.